<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartWindow</class>
 <widget class="QMainWindow" name="StartWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>381</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>34</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Mini Minecraft</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>161</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>New Game</string>
    </property>
    <widget class="QPushButton" name="newGameStart">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>New Game</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="newGameName">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>113</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="newGameSeed">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Game Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Seed</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>120</y>
      <width>251</width>
      <height>341</height>
     </rect>
    </property>
    <property name="title">
     <string>Load Game</string>
    </property>
    <widget class="QPushButton" name="loadGameStart">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>290</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Game</string>
     </property>
    </widget>
    <widget class="QListWidget" name="loadGameList">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>181</width>
       <height>241</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>560</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
